### DISCUSSION TITLE
Not able to use useSetAtom

### DISCUSSION THREAD

**Discussion by vasucp1207** [pmndrs/jotai#1707] [Q&A] [answered]
Encounter an unexpected error while using **useSetAtom**. [Sandbox](https://codesandbox.io/s/react-typescript-forked-kd4xi8) link.

**Comment by dai-shi** [comment, marked as answer]
useSetAtom is exported from `'jotai'`, not `'jotai/utils'`.

**Reply by vasucp1207** [reply]
Not work either,
<img width="506" alt="Screenshot 2023-01-22 at 11 59 22 PM" src="https://user-images.githubusercontent.com/85363195/213933366-4a5d4914-792d-4981-a82b-9b62117ae511.png">

**Reply by dai-shi** [reply]
You are using old version of jotai.
`useSetAtom` is introduced in https://github.com/pmndrs/jotai/releases/tag/v1.6.0.
